### ISSUE TITLE
Not working with Android 9 emulator

### ISSUE THREAD

**Issue by sanketplus** [square/picasso#1938]
I found it to be because of selinux enabled by default with newer androids. The debug error/info I see is

```
W/so-/DvpvklR.png: type=1400 audit(0.0:59): avc: denied { write } for name="perfd" dev="dm-0" ino=65545 scontext=u:r:untrusted_app:s0:c88,c256,c512,c768 tcontext=u:object_r:shell_data_file:s0 tclass=dir permissive=0
``` 

I tried to launch Emulator with selinux disabled with `emulator @Nexus_5X_API_28 -selinux disabled` but it's still `Enforced`. The same project working fine on my S8 though.

PS: trying out hello world picasso example. 

Happy to give more info.

Thanks!

**Comment by jrodbx** [comment]
The samples work for me on a Nexus 5 API 28 stock Google emulator.  Either way, I'm not certain this is a Picasso bug.  Please reopen if you have more information.

**Comment by NightlyNexus** [comment]
Fixed by #1967. Try the master branch now.
